src/components/Country.js

```diff
@@ -0,0 +1,101 @@
+import React, { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { FaClock } from "react-icons/fa";
+import { fetchCountries } from "../redux/countrySlice";
+import DateComponent from "../globals";
+
+const Country = () => {
+  const { countries, loading, error } = useSelector(
+    (state) => state.countrySlice
+  );
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCountries());
+  }, [dispatch]);
+  return (
+    <div>
+      {loading && <h3 className="text-center">Loading...</h3>}
+      {error && <h3>{error}</h3>}
+      <div className="grid grid-cols-2 mx-5 md:gap-10 overflow-hidden">
+        {countries.map((country) => {
+          return (
+            <div
+              className="h-max p-3 border-2 shadow-lg rounded bg-white flex flex-col justify-around"
+              key={country.country}
+            >
+              <div className="flex gap-3">
+                <div className="hidden bg-slate-200 h-32 lg:h-max lg:mb-5 w-1/3 md:flex items-center justify-center rounded p-3">
+                  <img
+                    // width={200}
+                    className="rounded"
+                    alt="Country's Flag"
+                    src={country.countryInfo.flag}
+                  />
+                </div>
+                <div className="text-left h-40">
+                  <div className="text-center md:text-left md:flex mb-5">
```

Comment:
As mentioned in [Project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#project-requirements)

> #### Interactions
> 
> **Home page**
> 
> When the page loads, the webapp shows the list of categories that could be filtered out by some parameter, for example by the category name.
> Along with the category name, you will display some numeric values per category that come from the API.
> When the user clicks (or taps) on a category item, the application navigates to the details page.
> 
> **Details page**
> 
> In the details page, the webapp retrieves data from the API to show detailed data in the category.
> When the user clicks on the "Back" button (<), the user navigates to the home page.
> 
> ...
>
>  Every page (the main page and the pages for each item) should have a unique route within the SPA

- [ ] Great job on dividing the cards into two columns however as suggested by all previous reviewers please display details of each card on a separate page and not on the same page. The first page should only display minimum data like country name and some way to click it so that upon clicking the card the user navigates to the details page of the selected card only.  ☝️ ☝️ ☝️ See the requirements.